abn/service.go

```diff
@@ -0,0 +1,142 @@
+// Package abn contains the implementation of the A/B/n service (via gRPC)
+package abn
+
+// service.go - entry point for A/B/n service
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	"golang.org/x/sys/unix"
+	"google.golang.org/grpc"
+	"google.golang.org/protobuf/types/known/emptypb"
+
+	"github.com/dgraph-io/badger/v4"
+	pb "github.com/iter8-tools/iter8/abn/grpc"
+	"github.com/iter8-tools/iter8/base/log"
+	"github.com/iter8-tools/iter8/controllers"
+	"github.com/iter8-tools/iter8/storage"
+	"github.com/iter8-tools/iter8/storage/badgerdb"
+
+	// auth package is necessary to enable authentication with various cloud providers
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+)
+
+const (
+	// defaultMetricsPath is the default path of the persistent volume
+	defaultMetricsPath = "/metrics"
```

Comment:
Seems to be in the wrong place? `defaultMetricsPath` is specific to storage client implementation. If at all, it belongs in `simple.go` of `badgerDB` package.

Reply:
Also, within for the storage implementation, this is the type of stuff we will want a config file to specify. Not hardcode in Golang.

Reply:
fixed